plugins/inputs/sqlserver/README.md

```diff
@@ -6,7 +6,7 @@ lightweight and use Dynamic Management Views supplied by SQL Server.
 
```

Comment:
We might add a section which lists the minimum requirements or supported editions for the plugin, something like:
```md
### THe SQL Server plugin supports the following editions/versions of SQL Server
- SQL Server
  - 2008 SP3 (with CU3)
  - SQL Server 2008 R2 SP2
  - any newer version
- Azure SQL DB
- Azure Managed Instance
```

Reply:
Done